playbook/app/pb_kits/playbook/pb_table/_table.scss

```diff
@@ -4,22 +4,62 @@
 $pb_table_row_kit_side_highlight_colors: map-merge(map-merge($status_colors, $product_colors), $category_colors);
 
 @mixin pb_table_row_kit_side_highlight($background) {
-    box-shadow: inset 4px 0 0 0 $background;
-    border-top: 0px;
+  box-shadow: inset 4px 0 0 0 $background;
+  border-top: 0px;
 }
 
 [class^=pb_table] {
-    tbody {
-        [class^=pb_table_row_kit] {
-            @each $color_name, $color_value in $pb_table_row_kit_side_highlight_colors {
-                &[class*=_side_highlight_#{$color_name}] {
-                    td {
-                        &:first-child {
-                            @include pb_table_row_kit_side_highlight($color_value);
-                        }
-                    }
-                }
+  tbody {
+    [class^=pb_table_row_kit] {
+      @each $color_name, $color_value in $pb_table_row_kit_side_highlight_colors {
+        &[class*=_side_highlight_#{$color_name}] {
+          td {
+            &:first-child {
+              @include pb_table_row_kit_side_highlight($color_value);
             }
+          }
         }
+      }
     }
-}
\ No newline at end of file
+  }
+  &.table-sm thead tr[align=center] th,
+  &.table-md thead tr[align=center] th,
+  &.table-lg thead tr[align=center] th,
+  &.table-sm thead tr th[align=center],
+  &.table-md thead tr th[align=center],
+  &.table-lg thead tr th[align=center] {
```

Comment:
Still a little confused by these selectors.  Feels like you may not need all of these.  What are you actually targeting here, the rows?

Reply:
If you need to override some competing specificity with other selectors there are other options.

Reply:
I agree, maybe even consider swapping utility css classes (I think they exist for alignment) within the rails/react.

1-2 lines in ruby/rails might get rid of all of these plus leverage existing classes/patterns.

Reply:
yeah align/ it'sfunction already exist so tr[align=center] in table body works without having to specify css but tr[align=center] was being overridden specifically in the table header. 
this is from inspecting/ the pre-existing CSS:
![Screen Shot 2020-11-12 at 12 57 33 PM](https://user-images.githubusercontent.com/64422843/98977520-b0124f00-24e6-11eb-9afb-0f2245e9156d.png)
and i didnt wanna change it because i wanted to keep the default as left aligned

Reply:
If you're targeting `th[align=center]` or `tr[align=center]` it won't override the default text-alignment.  Your styles will be limited to tags with `align=center`, unless I'm misunderstanding.

Reply:
"If you're targeting th[align=center] or tr[align=center] it won't override the default text-alignment" yes and because it wont override, i needed to use`[class^=pb_table].table-sm thead tr th[align=center]` as the CSS selctor/ target to override, so the align function works for the row specifically in my header if a user wants to implement the align functionality... does that make sense? because align is built in, it works for the rows within the body but thats because we havent fooled around with that at all. but since in our header we targeted `[class^=pb_table].table-sm thead tr th` and set it to `text-align: left`, tr[align=center], or th[align=center], was being overridden (but only overidden in the header)

Reply:
"If you need to override some competing specificity with other selectors there are other options." i did need to, and so i may have taken the long way of doing so.